src/Dropdown.php

```diff
@@ -3969,6 +3970,14 @@ public static function getDropdownActors($post, $json = true)
             }
         }
 
+        // permits hooks to alter actors list
+        $hook_results = Plugin::doHookFunction(Hooks::FILTER_ACTORS, [
+            'actors' => $results,
+            'input' => $post,
```

Comment:
Renaming have to be reverted, or documentation and implementation in example and escalade plugins have to be fixed.
```suggestion
            'params' => $post,
```